cmd/opentelemetry/cmd/all-in-one/main.go

```diff
@@ -106,7 +108,7 @@ func main() {
 	)
 
 	// parse flags to propagate Jaeger config file flag value to viper
-	cmd.ParseFlags(os.Args)
+	parseErr := cmd.ParseFlags(os.Args)
 	err = jflags.TryLoadConfigFile(v)
 	if err != nil {
 		handleErr(fmt.Errorf("could not load Jaeger configuration file %w", err))
```

Comment:
```suggestion
	if err := jflags.TryLoadConfigFile(v); err != nil {
		handleErr(fmt.Errorf("could not load Jaeger configuration file: %w", err))
```

Reply:
I didn't change this code because it doesn't solve directly the problem. 
it should have been in the related review.